docs/en/tutorials/add-contracts-pallet.html

```diff
@@ -186,11 +186,11 @@ <h3><a class="anchor" aria-hidden="true" id="implementing-the-contract-trait"></
 
 <span class="token comment">/*** Add This Block ***/</span>
 <span class="token macro property">parameter_types!</span> <span class="token punctuation">{</span>
-    <span class="token keyword">pub</span> <span class="token keyword">const</span> <span class="token class-name">TombstoneDeposit</span><span class="token punctuation">:</span> <span class="token class-name">Balance</span> <span class="token operator">=</span> <span class="token function">deposit</span><span class="token punctuation">(</span>
+    <span class="token keyword">pub</span> <span class="token class-name">TombstoneDeposit</span><span class="token punctuation">:</span> <span class="token class-name">Balance</span> <span class="token operator">=</span> <span class="token function">deposit</span><span class="token punctuation">(</span>
```

Comment:
Am I missing something? This document is rendered from the MD here right?
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/blob/source/docs/tutorials/add-contracts-pallet.md

If so, please change this file, and check it renders locally :pray:

Reply:
Ok source branch - working with md seems a lot nicer.